helpers/bignum.js

```diff
@@ -70,8 +70,8 @@ BigNumber.prototype.toBuffer = function(opts) {
 		}
 
 		var abs = this.abs();
-		var buf = abs.toBuffer({ size: 1, endian: 'big' });
-		var len = buf.length === 1 && buf[0] === 0 ? 0 : buf.length;
+		const buf = abs.toBuffer({ size: 1, endian: 'big' });
+		let len = buf.length === 1 && buf[0] === 0 ? 0 : buf.length;
```

Comment:
It's strange to see `var`, `const`, and `let` together like this. How about declaring these as `var`s at the top of the function and reassigning both here and further down? Then at some point we can switch to `const` and `let` altogether if that makes sense.